changelog.d/7043.wip

```diff
@@ -0,0 +1 @@
+[Devices Management] Refactor some code to improve testability
```

Comment:
This file should be removed, it was part of a previous work.

Reply:
Done.